driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverRequest.java

```diff
@@ -112,26 +113,20 @@ public void setWindowSize(Dimension dimension) {
     }
 
     private Dimension readDimensionFromString(final String windowSizeProperty) {
+        Dimension dimension = new Dimension(1920, 1080);
```

Comment:
Should this not be read from `Properties.WINDOW_SIZE.getDefault()` (or the other way around, or both from a Single Source of Truth)?

Reply:
Fixed.